crates/pink-drivers/system/lib.rs

```diff
@@ -10,21 +10,41 @@ pub use system::System;
 mod system {
     use super::pink;
     use alloc::string::String;
+    use alloc::vec::Vec;
     use ink::{codegen::Env, storage::Mapping};
     use pink::system::{CodeType, ContractDeposit, ContractDepositRef, DriverError, Error, Result};
     use pink::{HookPoint, PinkEnvironment};
 
     use this_crate::{version_tuple, VersionTuple};
 
+    /// A new driver is set.
+    #[ink(event)]
+    pub struct DriverChanged {
+        #[ink(topic)]
+        name: String,
+        previous: Option<AccountId>,
+        current: AccountId,
+    }
+
+    /// A new administrator is added.
```

Comment:
Why do you add a few events in this change?

Reply:
Since we may land the feature `Event chain`(https://github.com/Phala-Network/phala-blockchain/pull/1304), it should make sense to emit events in contracts.
They should be in a separate PR. But since the adding of `DriverChanged` would conflict with this PR, I put all of them in this PR.